### ISSUE TITLE
Prescan files

### ISSUE THREAD

**Issue by knochenhans** [emoon/HippoPlayer#123]
Currently, HippoPlayer only reads and displays file meta data only when it actually starts playing a file. It would be more practical if the player would scan files as soon as they are added (as, for example, DeliPlayer does). This would also help to identify unplayable files.

**Comment by emoon** [comment]
Yeah. The reason I haven't implemented this yet is because I have to verify the thread-safety of the players. Some of them aren't thread-safe so if you are trying to open a new song (to get metadata) it may screw up the current playing song.
